docs/setuptools.txt

```diff
@@ -1651,29 +1651,34 @@ See the sections below on the `egg_info`_ and `alias`_ commands for more ideas.
 
 
 
-Distributing Extensions compiled with Pyrex
--------------------------------------------
+Distributing Extensions compiled with Cython
+--------------------------------------------
 
-``setuptools`` includes transparent support for building Pyrex extensions, as
-long as you define your extensions using ``setuptools.Extension``, *not*
-``distutils.Extension``.  You must also not import anything from Pyrex in
-your setup script.
+``setuptools`` will detect at build time whether Cython is installed or not.
+If Cython is not found ``setputools`` will ignore pyx files. In case it's
+available you are supposed it will work with just a couple of adjustments.
+``setuptools`` includes transparent support for building Cython extensions, as
+long as you define your extensions using ``setuptools.Extension``.
+Then you should use Cython own ``build_ext`` in ``cmdclass``, e.g.::
+
+   from Cython.Distutils import build_ext
```

Comment:
I can never keep track, is this still the right way to do it?

I think that there's a way to do this that is more compatible with `setup_requires`, for whatever that's worth.

I'm fine with requiring PEP 518, but if we're going to have code snippets that only meaningfully work with PEP 518, we need to mention how to use `pyproject.toml` as well I think.

@jaraco Thoughts?